xkcd/cmd/main.go

```diff
@@ -0,0 +1,42 @@
+// xkcd is an xkcd offline indexing and quering tool.
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/vespian/go-exercises/xkcd/pkg/cmdline"
+	"github.com/vespian/go-exercises/xkcd/pkg/index"
+	"github.com/vespian/go-exercises/xkcd/pkg/types"
+)
+
+func main() {
+	var err error
+
+	c := cmdline.Parse()
+	fmt.Printf("Commandline: %+v", c)
```

Comment:
you likely want to have a '\n' at the end as printf doesn't add a newline automatically.